executor/slow_query.go

```diff
@@ -243,8 +279,17 @@ type offset struct {
 	length int
 }
 
-func (e *slowQueryRetriever) getBatchLog(reader *bufio.Reader, offset *offset, num int) ([]string, error) {
+type slowLogTask struct {
+	resultCh chan parsedSlowLog
+}
+
+type slowLogs struct {
+	log []string
+}
```

Comment:
How about use `type slowLogBlock = []string`

Reply:
Do you mean renaming `slowLogs` to `slowLogBlock`?